src/util/system.cpp

```diff
@@ -965,6 +967,19 @@ bool ArgsManager::ReadConfigStream(std::istream& stream, const std::string& file
     return true;
 }
 
+void ArgsManager::GetConfigFilePath(fs::path& conf_path, fs::path& conf_file)
+{
+    conf_path = GetPathArg("-conf", BITCOIN_CONF_FILENAME);
+    conf_file = GetConfigFile(conf_path);
+}
+
+void ArgsManager::GetOrCreateConfigFilePath()
```

Comment:
I'd suggest renaming this function too, it returns `void` so it's not really a "getter"

Reply:
Now returns a file path